### PR TITLE
Fixes for marvin drones

### PR DESCRIPTION
**Pull request by catornot**
Sometimes in sp worker drones spawn as error.mdl
since errors don't have anims a check must be added

### CODE REVIEW

File: Northstar.CustomServers/mod/scripts/vscripts/ai/_ai_marvin_jobs.gnut [hunk 1]
```diff
@@ -273,11 +273,14 @@ void function SimpleJobAnims( entity marvin, MarvinJob job )
...
 
+	if ( marvin.GetModelName() == $"models/error.mdl" )
+		return
+
 	while ( true )
 	{
 		anims.randomize()
 		foreach ( string anim in anims )
-		{
+		{
```
Comment (GeckoEidechse) [inline Northstar.CustomServers/mod/scripts/vscripts/ai/_ai_marvin_jobs.gnut:283]: Random whitespace added <.<

**Comment by GeckoEidechse** [comment]
So that I better understand the change the PR does, it basically just exits early if Marvin model couldn't be loaded?

**Comment by catornot** [comment]
yeah
in sp ( with co-op ) worker drones sometimes spawn without a model.

**Review by ASpoonPlaysGames — COMMENTED** [review]
Would be nicer to find out why they are spawning without the correct model than to add these checks, but this is fine for now. Could probably revert this commit once the underlying issue is resolved?

**Review by F1F7Y — APPROVED** [review]
Agree on Spoon with this. Would be nice to gget to the bottom of this issue, but as a temp fix this is good enough. Didnt test in-game

**Comment by catornot** [comment]
not needed anymore

**Comment by GeckoEidechse** [comment]
Out of the loop, what changed?

**Comment by catornot** [comment]
co-op is not getting merged soon because I want to completely rewrite it someday:tm: so might fiix it self or not :trollface:

**Pull request closed without merging**